Modules/_sqlite/connection.c

```diff
@@ -1866,6 +1866,53 @@ pysqlite_connection_exit_impl(pysqlite_Connection *self, PyObject *exc_type,
     Py_RETURN_FALSE;
 }
 
+/*[clinic input]
+_sqlite3.Connection.setlimit as setlimit
+
+    limit: int
+        The limit category to be set.
+    value: int
+        The new limit. If the new limit is a negative number, the limit is
+        unchanged.
+    /
+
+Set connection run-time limits. Non-standard.
```

Comment:
What does it mean "Non-standard"?

Reply:
I would prefer dropping "Non-standard" if you agree.

Reply:
Resolved in 1c36d9580d87a06e876dd27a21589cdba6fd952e